lib/geojson2image.rb

```diff
@@ -205,18 +235,32 @@ def draw(json, properties = nil)
           end
         end
 
-        border_points = []
-        json['coordinates'].each do |linestrings|
+        holes = []
+
+        json['coordinates'].each_with_index do |linestrings, position|
+          border_points = []
+
           if linestrings[0] != linestrings[linestrings.count - 1]
             linestrings << linestrings[0]
           end
+
           linestrings.each do |point|
             new_point = transform_point(point)
             border_points << "#{new_point[0]},#{new_point[1]}"
           end
+
+          points = ChunkyPNG::Vector.multiple_from_string(border_points.join(", "))
+
+          if position > 0
+            background = ChunkyPNG::Color::TRANSPARENT
+            tmp_fill = @background
+            holes << ChunkyPNG::Image.new(@width, @height, background).polygon(points, tmp_stroke, tmp_fill)
+          else
+            @png.polygon(points, tmp_stroke, tmp_fill)
+          end
+
+          holes.each { |hole| @png.compose! hole  }if holes.any?
```

Comment:
This is going to compose the collection of holes for each hole (position > 0) in the coordinates array. Did you mean to move it outside of the `.each_with_index`? Also, a couple of minor syntax changes.

```suggestion
          holes.each { |hole| @png.compose!(hole) } if holes.any?
```

Reply:
Yes, mean that.
But now I have a better Idea.